recipe/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Is this license shipped with openfst? If yes, we can use that one instead.

Reply:
Yes, same one. Does that mean I should just delete this file? And if so what if anything should I do to `meta.yaml`?

Reply:
1. yes, please delete it.
2. I don't think it will need a change.